ConsoleTest/ConsoleTest.csproj

```diff
@@ -107,6 +127,8 @@
     </ProjectReference>
   </ItemGroup>
   <ItemGroup>
+    <Content Include="Dependencies\x86_64\fmod64.dll" />
```

Comment:
The FMOD license doesn't allow redistributing the binaries. Could you replace this with a installation note and an installation script in the same way as the [x86](https://github.com/martindevans/SupersonicSound/tree/master/ConsoleTest/Dependencies/x86) folder is set up.